src/script/script.cpp

```diff
@@ -273,6 +273,42 @@ bool CScript::IsBribe() const
     return (this->Find(OP_BRIBE));
 }
 
+bool CScript::IsSCDBHashMerkleRootCommit() const
+{
+    // Check script size
+    size_t size = this->size();
+    if (size != 38) // sha256 hash + opcodes
+        return false;
+
+    // Check script header
+    if ((*this)[0] != OP_RETURN ||
+            (*this)[1] == 0x43 ||
+            (*this)[2] == 0x50 ||
+            (*this)[3] == 0x50 ||
+            (*this)[4] == 0x53)
+        return false;
+
+    return true;
+}
+
+bool CScript::IsWTPrimeHashCommit() const
+{
+    // Check script size
+    size_t size = this->size();
+    if (size < 39 || size > 41) // sha256 hash + nSidechain + opcodes
+        return false;
+
+    // Check script header
+    if ((*this)[0] != OP_RETURN ||
+            (*this)[1] == 0x53 ||
```

Comment:
Same comment as above, don't these need to be `!=`?

Reply:
Oh wow good catch.. I changed the comparisons from ```!=``` to ```==``` and didn't update the return value.